www/index.html

```diff
@@ -163,21 +163,22 @@ <h3 class="track-title">Arts</h3>
 				<a target="_blank" href="http://www.embeddedplanet.com"><img id="sponsors_eb" src="sponsor-logos/embedded-planet.jpg"></a>
 				<a target="_blank" href="https://www.ibm.com/watson/health/explorys/"><img id="sponsors_explorys" src="sponsor-logos/Explorys.svg"></a>
 			</div>
-			<div style="text-align:center; font-size:22px;">
-				<p>Learn more about sponsoring HackCWRU<br>Interested?  Email <a class="email_link" href="mailto:sxc868@case.edu">Sacheth</a>!</p>
-			</div>
 		</div>
-		<footer>
-			<div id="contact" class="footer">
-				<div class="copyr">
+		<div id="contact">
```

Comment:
My reasoning was:
- The "contact" id now encapsulates a different set of data so the footer tag seemed inappropriate.
- The footer class rather than the tag was adjusting the layout.

Is their any benefit to using a footer tag (not an experienced web dev)?